acceptance/api/v1/application_show_test.go

```diff
@@ -40,11 +41,17 @@ var _ = Describe("AppShow Endpoint", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(createdAt.Unix()).To(BeNumerically("<", time.Now().Unix()))
 
-		Expect(appObj.Workload.Restarts).To(BeNumerically("==", 0))
-
 		Expect(appObj.Workload.DesiredReplicas).To(BeNumerically("==", 1))
 		Expect(appObj.Workload.ReadyReplicas).To(BeNumerically("==", 1))
 
+		Expect(len(appObj.Workload.Replicas)).To(Equal(1))
+		var replica *models.PodInfo
+		for _, v := range appObj.Workload.Replicas {
```

Comment:
I realize now that we use the terms `Replica`, `Pod` and `Instance` in various places and we refer to the same thing. I will change it to be `Instance` everywhere (as in "application instance")